BUILD.gn

```diff
@@ -398,6 +398,7 @@ static_library("electron_lib") {
   }
   if (is_win) {
     libs += [ "dwmapi.lib" ]
+    cflags = [ "-Wno-microsoft-include" ]
```

Comment:
If this is a problem with node's headers, then this flag should go in the node GN files.

Reply:
@nornagon I tried that, it didn't help.  The issue comes from Electron including a node header which then causes this error rather than the node build itself.

Reply:
That's what `public_configs` is for in GN.